cdn/signUrl/README.md

```diff
@@ -0,0 +1,55 @@
+# Google Cloud Signing URL Sample
+
+This sample shows how to create Cloud CDN signed Url.
```

Comment:
Many readers will not know what CDN means.  Define it here.  Rename the directory something longer than cdn/.  Add a link to the documentation page.

Reply:
Agree with Ahmet on the folder, and agree with Jeff that we should add a comment on what CDN means with link. Adding a comment.